build.sh

```diff
@@ -165,7 +165,7 @@ fi
 
 pushd BuildLinux > /dev/null
 
-build_directory="${BUILD_TYPE,,}"
+build_directory="${BUILD_TYPE:0}"
```

Comment:
Tested and original code converts to lower case... Your change is equivalent to `build_directory=$BUILD_TYPE`? Was your goal to change the dir name case?